lib/storage/storage-init.js

```diff
@@ -48,8 +49,11 @@ function initStore(dataSetId, seedData, mediator) {
         return dataStore;
       });
     }).then(function(dataStore) {
-      //Initialisation complete, subscribe to any mediator topics
-      return dataStore.listen(dataTopicPrefix, mediator);
+      if (customListenHandler) {
```

Comment:
That's required to do override above. It's just enabling some specific datasets to define overrides. I can still do that globally but I do not think that's good idea, considering that filters should be applied only to workorders.